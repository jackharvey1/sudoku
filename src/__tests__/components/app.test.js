import React from 'react';
import App from '../../components/app';
import ShallowRenderer from 'react-test-renderer/shallow';

let renderer;

beforeEach(() => {
    renderer = new ShallowRenderer();
});

it('renders the App component as intended', () => {
    renderer.render(<App />);
    const output = renderer.getRenderOutput();
    expect(output).toMatchSnapshot();
});
