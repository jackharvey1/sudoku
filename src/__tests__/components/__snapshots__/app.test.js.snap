// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the App component as intended 1`] = `
<div>
  <Sudoku />
</div>
`;
